core/app/models/spree/user_return_to_storer.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Spree
+  # This service object is responsible for storing the current path into
+  # into `session[:spree_user_return_to]` for redirects after successful
+  # user/admin authentication.
+  class UserReturnToStorer
+    class Configuration
+      def rules
+        @rules ||= ::Spree::Core::ClassConstantizer::Set.new.tap do |set|
+          set << 'Spree::UserReturnToStorer::Rules::AuthenticationRule'
```

Comment:
I think it could be better setting the default rule into `core/lib/spree/app_configuration.rb` [where we are creating the preference](https://github.com/solidusio/solidus/pull/3369/files#diff-3c6f02645e665491dd8ba3931be30b9fR480-R482) to better split the implementation and how we are actually using it in Solidus and also for consistency with the other preferences that seem to have defaults set there.

Reply:
Sounds good 👍